osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -173,6 +181,14 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
                 time.sleep(5)
                 wait_time += 5
                 if wait_time > 30:
+                    logger.exception(
+                        "Even after waiting for #{wait_time} #{time_unit_abbreviation}, the extraction order "
+                        "#{extraction_order_id} hasn't been created".format(
```

Comment:
Uh, did you test this? The use of `#{...}` in [my comment](https://github.com/geometalab/osmaxx/pull/208#discussion_r40445305) was Ruby-inspired. It seems like [Python's `str.format()`](https://docs.python.org/3.4/library/stdtypes.html#str.format) uses [curly braces without the `#`](https://docs.python.org/3.4/library/string.html#formatstrings).